internal/uvm/start.go

```diff
@@ -199,12 +232,25 @@ func (uvm *UtilityVM) Start(ctx context.Context) (err error) {
 			Log:      log.G(ctx).WithField(logfields.UVMID, uvm.id),
 			IoListen: gcs.HvsockIoListen(uvm.runtimeID),
 		}
-		uvm.gc, err = gcc.Connect(ctx)
+
+		if isClone {
+			uvm.gc, err = gcc.Reconnect(ctx)
+		} else {
+			uvm.gc, err = gcc.Connect(ctx)
+		}
+
```

Comment:
I also feel like we should remove this if/else statement in favor of like 
`uvm.gc, err = gcc.Connect(ctx, isClone)` or something. Thoughts?